src/vcGIS.cpp

```diff
@@ -98,6 +98,10 @@ void vcGIS_GetOrthonormalBasis(const udGeoZone &zone, udDouble3 localPosition, u
 
 udDoubleQuat vcGIS_GetQuaternion(const udGeoZone &zone, udDouble3 localPosition)
 {
+  //Flat projections just return the unit quaternion.
+  if (zone.srid != 4978)
```

Comment:
Your comment suggests this is an ECEF feature, so shouldn't this be `zone.projection != udGZPT_ECEF`? Otherwise, it's only working for a very specific ECEF SRID.

Reply:
You are correct. I've added support for other projections.